schemas/dbt_yml_files.json

```diff
@@ -634,6 +683,12 @@
           "name": {
             "type": "string"
           },
+          "constraints": {
+            "$ref": "#/$defs/array_of_strings"
+          },
+          "constraints_check": {
```

Comment:
What is this one?

Reply:
Went poking around on GitHub and I think this might have been replaced too? https://github.com/dbt-labs/dbt-core/issues/7066